content/hi/cloud_native_security.md

```diff
@@ -0,0 +1,25 @@
+--- 
+
+Title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
+Status: completed 
+Category: अवधारणा 
+
+--- 
+
+ 
+
+## यह क्या है 
+
+क्लाउड नेटिव सुरक्षा एक ऐसा दृष्टिकोण है जो सुरक्षा को [क्लाउड नेटिव एप्लिकेशन](/cloud_native_apps/) में बनाता है। यह सुनिश्चित करता है कि सुरक्षा विकास के संपूर्ण एप्लिकेशन जीवनचक्र का हिस्सा है। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने का प्रयास करती है, जबकि क्लाउड नेटिव वातावरण के विवरणों को अपनाती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। क्लाउड नेटिव सुरक्षा `DevSecOps` नामक अभ्यास से अत्यधिक संबंधित है। 

```

Comment:
```suggestion
 क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने के साथ-साथ क्लाउड नेटिव वातावरण के विवरणों के अनुकूल होने का प्रयास करती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। 
```
translation for while is `जबकि` but it doesn't fit the meaning. I've added a suggestion here. Please take a look and incorporate if it makes sense.

Reply:
Okay, make sense!!
 this space is necessary right ? at the starting of the sentence
```
 क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा मॉडल के समान मानकों को सुनिश्चित करने के साथ-साथ क्लाउड नेटिव वातावरण के विवरणों के अनुकूल होने का प्रयास करती है, अर्थात् तेजी से कोड परिवर्तन और अत्यधिक अल्पकालिक बुनियादी ढाँचा। 

```

Reply:
you can get rid of the whitespace!  must have been a copy/paste addition